### PR TITLE
Pin tox, and update it to latest

### PR DESCRIPTION
**Pull request by stephendonner**

### CODE REVIEW

**Comment by stephendonner** [comment]
(Note to self: if this is the right approach, do similar for https://github.com/mozilla-services/axe-selenium-python/blob/92e3ea4f826c7df5043ebe7da90e0feb768dc658/Dockerfile and potentially other projects.)

File: tox.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+tox==3.1.2
```
Comment (davehunt) [inline tox.txt]: Are you planning to add this to the pyup.io configuration?

Comment (stephendonner) [reply tox.txt]: Yes; thanks for asking/reminding!

Would you prefer I do that here or in a separate PR?  Happy to do either (guessing the former?)

Comment (stephendonner) [reply tox.txt]: Done; I ended up fixing the pyup.io config here.

**Comment by stephendonner** [comment]
@davehunt thanks for the reviews; this should be good to go now.  Mind giving it one more pass-through?  Thanks again!

File: .pyup.yml [hunk 2]
```diff
@@ -1 +1,6 @@
 schedule: "every week on monday"
+requirements:
+  - requirements/flake8.txt
+  - requirements/py27-tests.txt
+  - requirements/py36-tests.txt
```
Comment (davehunt) [inline .pyup.yml]: These are already discovered by pyup.io's [file search](https://pyup.io/docs/bot/config/#file-search) so there's no need to list them explicitly. To avoid needing to maintain this list, let's revert this change. In fact, we could simply move `tox.txt` to `requirements/tox.txt` and we wouldn't need to change this file at all.
